ddtrace/internal/telemetry/data.py

```diff
@@ -73,7 +73,7 @@ def get_host_info():
     global _host_info
     if _host_info is None:
         _host_info = {
-            "os": platform.platform(aliased=True, terse=True),
+            "os": sys.platform,
```

Comment:
`sys.platform` is not as verbose as platform.platform() but it's less expensive (It does not include the architecture and version which is probably fine)